api/go.mod

```diff
@@ -1,13 +1,13 @@
 module jester.noble.xyz/api
 
-go 1.22
+go 1.24
 
 require (
 	connectrpc.com/connect v1.18.1
-	google.golang.org/protobuf v1.36.2
+	google.golang.org/protobuf v1.36.4
```

Comment:
Let's make sure to update the version in [`buf.gen.yaml`](https://github.com/noble-assets/jester/blob/main/buf.gen.yaml#L7)

Reply:
Good eye 
[`e754f77` (#9)](https://github.com/noble-assets/jester/pull/9/commits/e754f771c0873559637fe70fceb0c43856f0b444)